compile-protobuffers.js

```diff
@@ -0,0 +1,49 @@
+import { exec } from "child_process";
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+
+let dir = path.dirname(fileURLToPath(import.meta.url));
+let out_dir = `${dir}/src/lib/proto`;
+
+if (!fs.existsSync(out_dir)) fs.mkdirSync(out_dir);
+
+async function main() {
+	let files = await readdir(`${dir}/Ecdar-ProtoBuf`);
+	await Promise.all(
+		files
+			.filter(name => name.match(/.*\.proto/g))
+			.map(file =>
+				runCmd(`
```

Comment:
Protoc-ts is a commandline utility so it needs to be called from a commandline

Reply:
I don't like that, but if it is the only option the so be it. Could you add a todo and describe the issue here? I think it would be good for future developers to know that they should see if it is possible to call this lib or a future better lib directly.